### PR TITLE
Dietpi-Software | FuguHub

### PR DESCRIPTION
**Pull request by ghost**
Added FuguHub -arm only.

### CODE REVIEW

File: dietpi/dietpi-software [hunk 1]
```diff
@@ -7035,6 +7044,32 @@ _EOF_
...
+
+			Banner_Installing
+
+			INSTALL_URL_ADDRESS='http://FuguHub.com/releases/raspberrypi/install.sh'
+
+			/DietPi/dietpi/func/check_connection "$INSTALL_URL_ADDRESS"
+
+			if (( $? == 0 )); then
+
+				wget -O FHinstall.sh $INSTALL_URL_ADDRESS
```
Comment (Fourdee) [inline dietpi/dietpi-software]: Please use quotes on ```"$INSTALL_URL_ADDRESS"``` to prevent it leaking to shell.

File: dietpi/dietpi-software [hunk 2]
```diff
@@ -11590,6 +11625,37 @@ _EOF_
...
 		fi
 
+		#-------------------------------------------------------------------
+		#FuguHub Config
+		INSTALLING_INDEX=161
+		if (( ${aSOFTWARE_INSTALL_STATE[$INSTALLING_INDEX]} == 1 )); then
+
+			# Setup Filestore DietPi appropriate
+			# IF already present just create symlink
+			if [ ! -f "/mnt/dietpi_userdata/fuguhub-data/" ]; then
```
Comment (Fourdee) [inline dietpi/dietpi-software]: Please replace all references of ```/mnt/dietpi_userdata``` with the global defined VAR:
```"$FP_DIETPI_USERDATA_DIRECTORY"``` eg:

```
"$FP_DIETPI_USERDATA_DIRECTORY"/fuguhub-data/
```

File: dietpi/dietpi-software [hunk 3]
```diff
@@ -11590,6 +11625,37 @@ _EOF_
...
+				# Move installed filestore to dietpi folder
+				mkdir /mnt/dietpi_userdata/fuguhub-data/
+				mv /home/bd/disk/*  /mnt/dietpi_userdata/fuguhub-data/
+				# Removed 'actual' folder to make way for symlink
+				rm -r /home/bd/disk
+				# Create symlink
+				ln -s /mnt/dietpi_userdata/fuguhub-data /home/bd/disk
+				# Set permissions
+				setfacl -R -m u:bd:rwx /mnt/dietpi_userdata/fuguhub-data/
```
Comment (Fourdee) [inline dietpi/dietpi-software]: All permissions on DietPi userdata dir, should be set here (Executes at end of installs):
https://github.com/Fourdee/DietPi/blob/master/dietpi/dietpi-software#L7642

Else, permissions may be overridden.

**Comment by ghost** [comment]
I messed up my branch  I merged branches the wrong way. So hang tight until I fix this.

**Comment by ghost** [comment]
Ok my branch is back to where it was before. Sorry about that merge mess.

**Review by Fourdee — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
@Fourdee - Done.

**Comment by Fourdee** [comment]
@techdabbler 

Legend, thank you 👍

**Pull request merged**